classlib/src/main/java/org/teavm/classlib/java/util/TPriorityQueue.java

```diff
@@ -112,6 +112,7 @@ public boolean offer(E e) {
         if (e == null) {
             throw new TNullPointerException();
         }
+        comparator.compare(e, e);
```

Comment:
What's the purpose of this call?

Reply:
If we add not consistent element to queue of size 0 or 1 - it successfully adds and then we have broken state.

Reply:
What do you mean by "not consistent element"?

Reply:
Ok, you're right, this check not needed.